src/main/java/com/zipcodewilmington/StringArrayUtils.java

```diff
@@ -42,31 +43,45 @@ public static String getSecondToLastElement(String[] array) {
      * @return true if the array contains the specified `value`
      */ // TODO
     public static boolean contains(String[] array, String value) {
-        return false;
+        return Arrays.asList(array).contains(value);
```

Comment:
Clean! 🚿 
Alternative could be to reuse your `getNumberOfOccurrences` method...
`return getNumberOfOccurrences(array, value) > 0`